### DISCUSSION TITLE
After updating lazygit, I don't seem to get a preview of changes in filter mode anymore

### DISCUSSION THREAD

**Discussion by elmehalawi** [jesseduffield/lazygit#3949] [Q&A]
I used to get a preview of the selected commit when in filter mode. After updating lazygit to the latest version (0.44.1), that no longer shows. Now, when I run `lazygit -f somefile`, I get a list of commits but no preview on the right hand side.

Was this feature removed? Was there some default that was changed? I tried looking through the issues and couldn't find anything.

**Comment by stefanhaller** [comment]
You must be upgrading from a pretty old version, this has changed in v0.40.0. It changed in this commit: 1712ad08ae, and it looks like it was intentional. Maybe @jesseduffield can chime in to explain why it was changed; I never use the `-f` option, so I have no opinion whether I would consider this an improvement or a regression.

In the meantime, simply press shift-Minus to go to "half-screen" mode.

**Reply by elmehalawi** [reply]
Yeah that was it! It would be really nice to be able to set this default, but pressing _ works for now.

My use case for this is I run this in a floating terminal in a vim buffer to see the history of the file in the buffer. That diff view is specifically why I'm using lazygit in this mode. If no one else has a similar use case, I'm satisfied using _ to switch to the view I want.
